### PR TITLE
fix: custom domain

### PR DESCRIPTION
**Pull request by owlet42**
Signed-off-by: ChenLong Ma <owlet42@126.com>

Fixes  ISSUE #336

### CODE REVIEW

File: helm-charts/FATE/values-template-example.yaml [hunk 1]
```diff
@@ -26,6 +26,12 @@ modules:
 
 backend: eggroll
```
Comment (jiahaoc1993) [inline helm-charts/FATE/values-template-example.yaml:28]: I think we could add some comments here, to make user know what do these commented line do.

File: helm-charts/FATE/values-template-example.yaml [hunk 2]
```diff
@@ -26,6 +26,12 @@ modules:
...
 
+# host:
+  # fateboard: 9999.fateboard.abc.net
```
Comment (jiahaoc1993) [inline helm-charts/FATE/values-template-example.yaml:30]: the party name should not be part of mutable host.

File: helm-charts/FATE/values-template-example.yaml (hunk 1 above)
Comment (owlet42) [reply helm-charts/FATE/values-template-example.yaml:28]: We can add features about adding comments

File: helm-charts/FATE/values-template-example.yaml (hunk 2 above)
Comment (owlet42) [reply helm-charts/FATE/values-template-example.yaml]: I think it's easier to understand

**Review by jiahaoc1993 — APPROVED** [review]

**Pull request merged**